@vates/types/src/xo.mts

```diff
@@ -44,7 +44,9 @@ type BaseXoVm = {
     process?: string
     version?: string
   }
+  // deprecated, use nestedVirt instead
```

Comment:
Use JSDoc, otherwise your IDE won't understand that the field is deprecated
```suggestion
  /**
   * @deprecated use isNestedVirtEnable instead
   */
```